src/main/java/de/tum/in/www1/artemis/service/connectors/lti/LtiService.java

```diff
@@ -112,11 +123,18 @@ private Authentication createNewUserFromLaunchRequest(String email, String usern
             final User newUser;
             final var groups = new HashSet<String>();
             groups.add(LTI_GROUP_NAME);
-            newUser = userCreationService.createUser(username, null, groups, firstName, lastName, email, null, null, Constants.DEFAULT_LANGUAGE, true);
+
+            var password = RandomUtil.generatePassword();
```

Comment:
I wouldn't use var here

Reply:
I used var because it's shorter and clearer in this case. But I get that it might be confusing. Why do you think I shouldn't use var here?

Reply:
For readability reasons (and according to our guidelines as far as I remember) `var` should be used to significantly shorten variable types. `String` is not a long type, though.